src/machine-config.h

```diff
@@ -117,6 +119,33 @@ struct htif_config final {
     bool yield_automatic{false};      ///< Make yield automatic available?
 };
 
+/// \brief VirtIO console device state config
+struct virtio_console_config final {};
+
+/// \brief VirtIO Plan 9 filesystem device state config
+struct virtio_p9fs_config final {
```

Comment:
9p?

Reply:
Have you looked at virtiofs? Is it simpler/harder, slower/faster?

Reply:
I would use "host_directory" instead of "shared_path".

Reply:
And just "tag" for the mount_tag, since this is what the virtio specification uses.

Reply:
> 9p or p9?

Both names are used in multiple project, the correct is 9P, but P9 is usually used in open source projects because in C names can't begin with numbers, so the code uses P9 for variable names, and file names, and class names.

> Have you looked at virtiofs? Is it simpler/harder, slower/faster?
Yes, I have. It is harder, because it requires an external application running in the host, however it is faster. I don't think it make sense for us to support it, because it's a Linux only solution. While 9P can work on any OS.